src/main/java/io/vertx/ext/dropwizard/impl/VertxMetricsImpl.java

```diff
@@ -156,30 +158,9 @@ public DatagramSocketMetrics createMetrics(DatagramSocket socket, DatagramSocket
   }
 
   @Override
-  public <P> PoolMetrics<?> createMetrics(P pool, String name, int size) {
-    // Todo
-    return new PoolMetrics<Object>() {
-      @Override
-      public Object taskSubmitted() {
-        return null;
-      }
-      @Override
-      public void taskRejected(Object task) {
-      }
-      @Override
-      public void taskBegin(Object task) {
-      }
-      @Override
-      public void taskEnd(Object task, boolean succeeded) {
-      }
-      @Override
-      public boolean isEnabled() {
-        return false;
-      }
-      @Override
-      public void close() {
-      }
-    };
+  public <P> PoolMetrics<?> createMetrics(P pool, String poolType, String poolName, int maxPoolSize) {
```

Comment:
Have you sent you changes to the SPI to vert.x core repo? This new poolType param is missing there (very nice addition by the way)